src/components/screens/multisigAccountDetails/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { withTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+import Box from '../../toolbox/box';
+import BoxHeader from '../../toolbox/box/header';
+import BoxContent from '../../toolbox/box/content';
+import BoxInfoText from '../../toolbox/box/infoText';
+import Dialog from '../../toolbox/dialog/dialog';
+import Tooltip from '../../toolbox/tooltip/tooltip';
+import { Members } from '../multiSignature/summary/summary';
+import { getActiveTokenAccount, extractAddress } from '../../../utils/account';
+import styles from './styles.css';
+
+const MultisigAccountDetails = ({ t }) => {
+  const account = useSelector(state => getActiveTokenAccount(state));
+
+  // @todo We shouldn't be needing this if we lazyload the routes.
+  if (!account.keys || account.keys.numberOfSignatures === 0) return null;
+
+  const { numberOfSignatures, optionalKeys, mandatoryKeys } = account.keys;
+  const [members, setMembers] = useState([]);
```

Comment:
Why do you need to store members in state? Can we not extract them directly from `account.keys...`?

Reply:
The cyprographical calculation of address from publicKey is a heavy duty. If we perform this on up to 64 member, it'll slow down the re-render significantly.

Reply:
In that case, you probably wanna use useMemo and calculate the members only when account.keys change.

Reply:
Done